definitions/TagGroup/com.adlinktech.vision.system/2.2.0/TrainingStreamerConfigTagGroup.json

```
@@ -0,0 +1,53 @@
+[
+    {
+        "name": "TrainingStreamerConfig",
+        "context": "com.adlinktech.vision.system",
+        "qosProfile": "state",
+        "version": "2.2.0",
+        "description": "Reports if a training streamer status is change.",
+        "tags": [
+            {
+                "name": "FrameStorage",
+                "description": "Save location in local or remote.",
+                "kind": "STRING",
+                "unit": "NA"
+            },
+            {
+                "name": "StreamId",
+                "description": "List of stream id",
```

The name and description don't seem right here. Maybe they should be
```suggestion
                "name": "StreamInformation",
                "description": "Information about the stream",
```

---

Changed. I also changed the description of the overall tag group.